### DISCUSSION TITLE
Google AdManager

### DISCUSSION THREAD

**Discussion by Handtaker23** [apache/airflow#44334] [Ideas]
**Description**

I would be able to provide a hook and operator to load reports from Google AdManager. In this issue I would like to know if there is interest in this?!

**Use case / motivation**

Dealing with ad platforms can be quite challenging because every platform has its own more or less well documented API. That is why I would like to provide an easy implementation to get reports from Google AdManager and to load the data into S3 or GCS.

**Related Issues**

None that I know.

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by mik-laj** [comment]
@Handtaker23 That sounds fantastic. I would be very interested in them. Let me know if you start work. I will be happy to help with reviews.

**Comment by kweitzner** [comment]
Very interested in this, I built my own based off Concorde airflow plug-in, but I have low experience with airflow right now.

**Comment by dephusluke** [comment]
@Handtaker23  I would be interested in AdManager hook.

**Comment by turbaszek** [comment]
@kweitzner @dephusluke @Handtaker23 do anyone of you would like work on this one? We are happy to help 👌

**Comment by Handtaker23** [comment]
@turbaszek I am happy to make my running hooks and operators for this "official airflow repo ready". I am using them already for many months in production. Time is my enemy at the moment. :(

**Comment by turbaszek** [comment]
@Handtaker23 it's always a good starting point but no pressure 👍

**Comment by bhavaniravi** [comment]
If this still has interest, I would like to take this up

**Comment by kweitzner** [comment]
Definitely interested

**Comment by eladkal** [comment]
assigned to you @bhavaniravi

**Comment by bhavaniravi** [comment]
It looks like ads to gcs are already available in [google ads provider](https://airflow.apache.org/docs/apache-airflow-providers-google/stable/_api/airflow/providers/google/ads/transfers/ads_to_gcs/index.html)

I will start working on the S3 integration

**Comment by eladkal** [comment]
@bhavaniravi are you still working on it?

**Comment by RoopaShastri** [comment]
Is there any update on this?
